### PR TITLE
Updated with v6.2 turbo-go-sdk and turbo-api packages

### PR DESCRIPTION
**Pull request by pallavidn**
Using godep updated the turbo-go-sdk and turbo-api packages with the v6.2 tags.

### CODE REVIEW

File: vendor/github.com/turbonomic/turbo-go-sdk/pkg/service/tap_service.go [hunk 1]
```diff
@@ -70,6 +70,12 @@ func (tapService *TAPService) addTarget(isRegistered chan bool) {
 
 func (tapService *TAPService) ConnectToTurbo() {
 	glog.V(4).Infof("[ConnectToTurbo] Enter ******* ")
+
+	if tapService.RESTClient == nil {
+		glog.V(1).Infof("TAP service cannot be started - cannot create target")
```
Comment (maxwangvmt) [inline vendor/github.com/turbonomic/turbo-go-sdk/pkg/service/tap_service.go]: It should be an error message, isn't it?

File: vendor/github.com/turbonomic/turbo-api/pkg/client/request.go [hunk 2]
```diff
@@ -183,10 +184,10 @@ func (r *Request) request(fn func(*http.Response)) error {
 			req.Header.Set(key, value)
 		}
 	}
-	// Set basic authentication for the request if there is any.
-	if r.basicAuth != nil {
-		req.SetBasicAuth(r.basicAuth.username, r.basicAuth.password)
-	}
+	//// Set basic authentication for the request if there is any.
```
Comment (maxwangvmt) [inline vendor/github.com/turbonomic/turbo-api/pkg/client/request.go]: Remove the commented code if it's not kept on purpose.

Comment (pallavidn) [reply vendor/github.com/turbonomic/turbo-api/pkg/client/request.go]: This is a change in the turbo-api and should be handled there.

File: vendor/github.com/turbonomic/turbo-go-sdk/pkg/service/tap_service.go (hunk 1 above)
Comment (pallavidn) [reply vendor/github.com/turbonomic/turbo-go-sdk/pkg/service/tap_service.go]: This is a change in the turbo-go-sdk and should be handled there.

**Pull request merged**